controllers/utils.go

```diff
@@ -31,21 +33,66 @@ func replaceProjectName(replacement string, m map[string]interface{}) {
 		case string:
 			s := m[k].(string)
 			m[k] = strings.ReplaceAll(s, "${PROJECT_NAME}", replacement)
+		case []string:
+			for i, s := range v.([]string) {
+				v.([]string)[i] = strings.ReplaceAll(s, "${PROJECT_NAME}", replacement)
+			}
 		case int64:
+		case []int64:
 		case int32:
+		case []int32:
 		case int:
+		case []int:
 		case bool:
+		case []bool:
 			continue
-		case []interface{}:
-			for _, elem := range v.([]interface{}) {
-				replaceProjectName(replacement, elem.(map[string]interface{}))
+		case []map[string]interface{}:
+			for _, a := range v.([]map[string]interface{}) {
+				replaceProjectName(replacement, a)
 			}
+		case []interface{}:
+			replaceProjectNameInSlice(replacement, m[k].([]interface{}))
```

Comment:
Wouldn't a `for` loop here be better suited to call `replaceProjectName` instead of having the almost identical `replaceProjectNameInSlice`?

Reply:
I think I've found a better way now